src/proctored-exam-settings/ProctoredExamSettings.jsx

```diff
@@ -87,11 +87,11 @@ function ProctoredExamSettings({ courseId, intl }) {
       proctored_exam_settings: {
         enable_proctored_exams: enableProctoredExams,
         proctoring_provider: proctoringProvider,
+        create_zendesk_tickets: createZendeskTickets,
```

Comment:
If we are now sending this field in the POST request, should we also make the field visible to non edX Staff? Or is it fine that they are unknowingly modifying this field?